dvc/command/plots.py

```diff
@@ -116,7 +116,7 @@ def add_parser(subparsers, parent_parser):
 
     fix_subparsers(plots_subparsers)
 
-    SHOW_HELP = "Generate a plots image file from a metrics file."
+    SHOW_HELP = "Generate plot from a metrics file."
```

Comment:
Actually, it can generate multiple plots from multiple metrics files.

Reply:
Fixed